src/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.blocks {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  flex-shrink: 0;
+  text-align: center;
+  line-height: 300px;
+}
+
+.block-1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width:600px) {
+  .block-3 {
+    order: 1;
+  }
+
+  .block-5 {
+    order: 2;
+  }
+
+  .block-6 {
+    order: 3;
+  }
+}
+
+@media (min-width:900px) {
+  .block-4 {
+    order: 3;
+  }
+
+  .block-5 {
+    order: 2;
+  }
+
+  .block-6 {
+    order: 1;
+  }
+}
+
+@media (min-width:1200px) {
+  .blocks {
+    max-width: 1199px;
+  }
+}
```

Comment:
don't hardcode conatiner width. there should be 3 blocks on any screen width > 900px. try to find a solution, using a `flex-basis` style. now your solution looks like this
![image](https://user-images.githubusercontent.com/37161315/167666695-e5df31f2-72fd-40b2-8196-5a5e47ce750e.png)
```suggestion
```